### PR TITLE
Doc: comment mips/proof.rs

### PR DESCRIPTION
**Pull request by dannywillems**

### CODE REVIEW

File: optimism/src/mips/proof.rs [hunk 1]
```diff
@@ -19,13 +19,15 @@ use rayon::iter::{
...
 
+/// Represents one line of the execution trace of the virtual machine
 #[derive(Debug)]
 pub struct WitnessColumns<G> {
     pub scratch: [G; crate::mips::witness::SCRATCH_SIZE],
     pub instruction_counter: G,
     pub error: G,
 }
 
+/// Helpers to iterate over the elements of the execution trace in parallel
```
Comment (mrmr1993) [inline optimism/src/mips/proof.rs:30]: Not sure that this adds any value; I'd lean towards closing this to keep noise down.

**Review by querolita — APPROVED** [review]
Maybe leave the doc comment for WitnessColumns but remove the ones for the iterators

File: optimism/src/mips/proof.rs (hunk 1 above)
Comment (dannywillems) [reply optimism/src/mips/proof.rs]: Removed

**Comment by dannywillems** [comment]
> Maybe leave the doc comment for WitnessColumns but remove the ones for the iterators

Done. I also added more documentation regarding the structure.

**Pull request merged**